cmd/algocfg/profileCommand.go

```diff
@@ -19,39 +19,81 @@ package main
 import (
 	"bufio"
 	"fmt"
-	"github.com/spf13/cobra"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/spf13/cobra"
+
 	"github.com/algorand/go-algorand/cmd/util/datadir"
 	"github.com/algorand/go-algorand/config"
 	"github.com/algorand/go-algorand/util/codecs"
 )
 
-// profileConfigUpdater updates the provided config for non-defaults in a given profile
-type profileConfigUpdater func(cfg config.Local) config.Local
-
-// defaultConfigUpdater leaves all default values in place
-func defaultConfigUpdater(cfg config.Local) config.Local {
-	return cfg
-}
-
-// relayConfigUpdater alters config values to set up a relay node
-func relayConfigUpdater(cfg config.Local) config.Local {
-	cfg.Archival = true
-	cfg.EnableLedgerService = true
-	cfg.EnableBlockService = true
-	cfg.NetAddress = "4160"
-	return cfg
+// configUpdater updates the provided config for non-defaults in a given profile
+type configUpdater struct {
+	updateFunc  func(cfg config.Local) config.Local
+	description string
 }
 
 var (
+	archival = configUpdater{
+		description: "Store all historic blocks for the API.",
```

Comment:
Could be something like:
"Stores the entire ledger to allow block serving and full API usage. Catchpoint Generation is turned off by default."?